server/command/install.go

```diff
@@ -92,11 +89,6 @@ func (s *service) executeExperimentalInstall(params *params) (*model.CommandResp
 		return normalOut(params, nil, err)
 	}
 
-	_, err = s.api.AWS.InvokeLambda(name, "_on_activate", lambda.InvocationTypeEvent, "some input")
-	if err != nil {
-		return normalOut(params, nil, err)
-	}
-
 	return &model.CommandResponse{
 		Text:         fmt.Sprintf("installed lambda functions from url %s.", releaseURL),
```

Comment:
to remove? Beyond this, do we still need `executeExperimentalInstall`

Reply:
@catalintomai Yes we do it installs lambda functions in the AWS account. Ideally this should be within the same flow. Install on aws, install/connect with `plugin_apps`. Now it is done by two different commands, because we are not yet clear about the  exact flow.

Reply:
Agree with @catalin, if this command is of no value ATM, remove it